sklearn/model_selection/tests/test_validation.py

```diff
@@ -1167,6 +1169,20 @@ def test_learning_curve_with_shuffle():
                               test_scores_batch.mean(axis=1))
 
 
+def test_learning_curve_fit_params():
+    est = SVC(random_state=0)
+    X, y = make_blobs(n_samples=50, n_features=2, centers=10, random_state=0)
+    w = np.arange(50)
+    gamma_range = np.logspace(-6, -1, 5)
+    l_with_samples_weight = validation_curve(est, X, y, "gamma", gamma_range,
+                                             fit_params={'sample_weight': w})
+    l_without_sample_weight = validation_curve(est, X, y, "gamma", gamma_range)
+    assert_not_equal(l_with_samples_weight[0].mean(),
```

Comment:
We also want to avoid almost-equality. `assert not np.isclose(x,y)` or `assert x != pytest.approx(y)`  might be a good solution.